SeaBattle/SeaBattle/GameBoard.cs

```diff
@@ -128,31 +123,35 @@ private bool CanAddShip(Ship s, int y, int x, Orientation orientation)
                     }
                     break;
             }
-          
-
             return true;
         }
 
 
         private void prepareCells()
         {
-            for (int i = 0; i < boardSize; i++)
-                for (int j = 0; j < boardSize; j++)
+            for (int i = 0; i < boardSize; i++) {
+                for (int j = 0; j < boardSize; j++) {
                     cells[i, j] = new Cell();
+                }
+            }
         }
 
 
         private void prepareShips()
         {
             int i;
-            for (i = 0; i < Settings.fourDeckersCount; i++)
+            for (i = 0; i < Settings.fourDeckersCount; i++) {

```

Comment:
It is C# not Java - move curly brackets to the next line

Reply:
P.S. Check all pr

Reply:
Of course, I will. Any whim for your money